playbook/hosts

```diff
@@ -2,9 +2,19 @@
 terminator.aelve.com
 guide.aelve.com
 s1.codesearch.aelve.com
+s1.staging.guide.aelve.com
+staging.guide.aelve.com
+
+[all:vars]
+ansible_python_interpreter=/usr/bin/python3
+port_main=8080
```

Comment:
Please clarify that those are related to Guide. We have Codesearch stuff in the same file.

(Frankly we should split that into several files, but for now let's keep it this way.)